TheCountBot.Application/TelegramBotManager.cs

```diff
@@ -219,10 +226,14 @@ private async Task HandleCoolNumbersAsync( int x, string user )
             {
                 await SendMessageAsync( $"@{user} blaze it!" );
             }
+            else if ( IsChaotic() ) 
+            {
+                await SendMessageAsync( $"AYO @{user} NICE WORK" );
```

Comment:
I like the chaos, but my first reaction to the text in this message was "is it sarcastic?"

thoughts on making it a bit more clear?

Reply:
🤔 I see what you mean about it being sarcastic.